docs/concepts/git/best-practices.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Best Practices
+id: git-best-practices
+slug: git-best-practices
+description: Learn about what we recommend to do if you are working with Git.
+tags:
+  - git
+  - best practices
+---
+
+## Overview
+
+To minimize and ideally avoid conflicts in your pull requests:
+
+1. Have each user create their own feature branches.
+1. Keep feature branches modular, which means to:
+
+   - Only modify assets as needed on your feature branch
+   - Avoid modifying assets that other users are modifying in parallel feature branches
+
+## Deep dive: Branching Strategy
+
+Use a proper Git branching strategy that aligns with your team's goals.
+
+### Small teams
+
+For small teams:
```

Comment:
For small teams, try the branching strategy used by GitHub and described in [this blog](https://www.alexhyett.com/git-flow-github-flow/#what-is-github-flow):